colossalai/inference/flash_decoding_utils.py

```diff
@@ -12,6 +12,11 @@ class FDIntermTensors(metaclass=SingletonMeta):
     def __init__(self):
         self._tensors_initialized = False
 
+    def _pre_initialize(self):
```

Comment:
What does `pre` mean? The implementation looks more like `reset` to me.

Reply:
I will rename this func